wdio.conf.js

```diff
@@ -254,7 +254,8 @@ const initSeleniumConfig = (isLocal, config = {}) => {
 
 const initBrowserStackConfig = (isLocal, config = {}) => {
   let mode = MODE_BROWSERSTACK;
-  let isCi = process.env.IS_CI !== "";
+  //let isCi = process.env.IS_CI !== "";
```

Comment:
Personally I would remove the old commented code in this file (line 257 and 296-329, leave the bits in the middle), I don't think it adds anything to leave them here, and it wouldn't be hard to reinstate the IS_CI check if it turns out it is required for something.

Reply:
Done